### PR TITLE
Do not ask twice for MFA

### PR DESCRIPTION
**Pull request by julienduchesne**
When an MFA is set, we just can't extend the session duration
I tried finding a way to read the config but there doesn't seem to be an API in the Go SDK

Also running tests on Windows too

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -209,12 +211,18 @@ func (config *TGFConfig) getAwsSession(duration int64) (*session.Session, error)
...
 			}
 		}
-		log.Warningf("Your AWS configuration is set to expire your session in %v (automatically extended to %v)",
-			duration,
-			time.Duration(maxDuration)*time.Second)
+		if askedForMfa {
+			log.Warningf("Your AWS configuration is set to expire your session in %v. This timeout could not be automatically extended due to the session's MFA",
+				duration)
+		} else {
+			log.Warningf("Your AWS configuration is set to expire your session in %v (automatically extended to %v)",
```
Comment (AbdulRahmanAlHamali) [inline config.go:218]: Could you make this `will extend by %v`,
and then after the next line you add a log saying `session extended successfully`?

**Review by AbdulRahmanAlHamali — APPROVED** [review]

File: config.go (hunk 1 above)
Comment (julienduchesne) [reply config.go:218]: Hey, I'm not done. How did you find this PR?

Comment (AbdulRahmanAlHamali) [reply config.go:218]: lol I don't know I was added as a reviewer :P

Comment (jocgir) [reply config.go:219]: In my last discussion with @JPLachance, we even discussed to put the MFA mandatory for all environments. So, that fix will be more than welcome.

**Pull request merged**